docs/orchestration/recipes/deployment.md

```diff
@@ -17,7 +17,7 @@ Docker provides an excellent industry-standard abstraction for shipping code alo
 Whenever you call `flow.register` or build a [Docker storage object](../../api/latest/environments/storage.html#docker) yourself, Prefect will perform the following actions:
 
 - calls `cloudpickle.dumps(flow)` on your Flow object to convert it to serialized bytes
-- stores these bytes inside the Docker image in the `/root/.prefect/` directory
+- stores these bytes inside the Docker image in the `/opt/.prefect/` directory
```

Comment:
Dot directories in `/opt/` feel a little off to me, perhaps `/opt/prefect`? Either works though.

Reply:
What are your thoughts on defaulting to something other than `opt`? I do like keeping with `.prefect` since that's what it is locally

Reply:
IMO dot directories only belong under `~/`, anything global shouldn't be hidden and should have an explicit name. Here `/opt/prefect`, or `/var/prefect` make the most sense to me (`/opt/prefect` I think makes more sense).

There is prior convention for this too - many systems load configuration/plugins from a mix of local and global directories (for example, dask looks in `~/.config/dask` then `/etc/dask`).

All of these are only conventions though, it doesn't really matter where we put these things as long as we're consistent.